### DISCUSSION TITLE
LDAP connection string

### DISCUSSION THREAD

**Discussion by hyh1618** [oracle/python-oracledb#95] [General]
I have to use LDAP connection string, like: "jdbc:oracle:thin:@ldap://host:port/cn=any,dc=any,dc=any/db".
Can any one please help me with this kind of connection in python-oracledb? or is it really not supported. 
Thanks.

**Comment by cjbj** [comment]
You'll need to (i) use python-oracledb Thick mode (ii) configure sqlnet.ora and ldap.ora files.  Check https://stackoverflow.com/questions/37933090/cx-oracle-ldap-connection-string-syntax/67604220#67604220 (which mentions cx_Oracle which was the older name for python-oracledb)

**Comment by cjbj** [comment]
Documentation for working with LDAP in Thick & Thin modes is at https://python-oracledb.readthedocs.io/en/latest/user_guide/connection_handling.html#ldap-directory-naming

See https://github.com/oracle/python-oracledb/issues/111
